src/discord_notifier.rs

```diff
@@ -89,6 +89,7 @@ impl DiscordNotifier {
 	}
 
 	pub async fn notify_users(&self, day: Weekdays, substitutions: &SubstitutionSchedule, users_to_notify: HashSet<u64>) -> Result<(), serenity::Error> {
+		log::debug!("Notifying users on discord");
```

Comment:
This is bloat, do not use this. Please implement your own programming language for this.